css/styles.css

```diff
@@ -29,8 +29,12 @@ div.crumb.last a {
 	width: 40px;
 }
 
-#controls > .right {
-	float: right;
+#controls .button:not(:last-child) {
```

Comment:
this should go to https://github.com/nextcloud/server/pull/7478

Reply:
Gallery uses a lot of similar styles but does not includes any of them (yes, we need to rebuild the entire app), therefor, there are a lot of multiples files/properties used.

Reply:
:cry: